main.js

```diff
@@ -0,0 +1,250 @@
+}
+
+console.log(isObject({}));
+console.log(isObject(10));
+console.log(isObject(null));
+
+// 4
+
+const userTwo = {
+    name: 'John'
+}
+
+function removeProperty(user, property) {
+    if (user.hasOwnProperty(property)) {
+        delete user[property];
+        return true;
+    }
+    return false;
+}
+
+console.log(removeProperty(userTwo, 'name'));
+console.log(removeProperty(userTwo, 'name'));
+console.log(removeProperty(userTwo, 'weight'));
+
+// 5
+
+const firstJohn = {
+    firstName: 'John',
+    lastName: 'Smith'
+}
+
+const secondJohn = {
+    firstName: 'John',
+    lastName: 'Smith'
+}
+
+const adam = {
+    firstName: 'Adam',
+    lastName: 'Johnson'
+}
+
+function checkIfUsersHaveTheSameName(objectOne, objectTwo) {
+    return objectOne.firstName === objectTwo.firstName
+        && objectOne.lastName === objectTwo.lastName;
+}
+
+console.log(checkIfUsersHaveTheSameName(firstJohn, secondJohn));
+console.log(checkIfUsersHaveTheSameName(firstJohn, adam));
+
+// 6
+
+function getVoteCount(votes) {
+    return votes.upvotes - votes.downvotes;
+}
+
+console.log(getVoteCount({ upvotes: 10, downvotes: 5 }));
+console.log(getVoteCount({ upvotes: 75, downvotes: 90 }));
+console.log(getVoteCount({ upvotes: 50, downvotes: 50 }));
+
+// 7
+
+function getCubeVolume(cube) {
+    return cube.height * cube.length * cube.width;
+}
+
+console.log(getCubeVolume({ width: 10, length: 5, height: 2 }));
+console.log(getCubeVolume({ width: 100, length: 500, height: 0 }));
+console.log(getCubeVolume({ width: 15, length: 2, height: 5 }));
+
+// 8
+
+function getCityInformation(city) {
+    return `${city.name} is in ${city.country} and has an area of ${city.areaInKilometers} square kilometers`;
+}
+
+console.log(getCityInformation({
+    name: 'Warsaw',
+    country: 'Poland',
+    areaInKilometers: 517
+}));
+
+// 9
+
+const john = {
+    firstName: 'John',
+    lastName: 'Smith'
+}
+
+function getUserCopy(user) {
+    return {
+        firstName: user.firstName,
+        lastName: user.lastName
+    }
+}
+
+const newUser = getUserCopy(john);
+console.log(newUser.firstName);
+console.log(newUser.lastName);
+console.log(newUser === john);
+
+// 10
+
+function getAnimalDescription(animal){
+    return `This ${animal.color} ${animal.name} has ${animal.legs} legs.`
+}
+
+// 11
+
+const rooms = {
+    first: {
+        description: 'This is the first room',
+        items: {
+            chair: 'The old chair looks comfortable',
+            lamp: 'This lamp looks ancient'
+        }
+    },
+    second: {
+        description: 'This is the second room',
+        items: {
+            couch: 'This couch looks like it would hurt your back',
+            table: 'On the table there is an unopened bottle of water'
+        }
+    }
+}
+
+// 12
+
+function addContact(contacts, name, phoneNumber) {
+    const newContacts = {...contacts};
+
+    newContacts[name] = {
+        name: name,
+        phoneNumber: phoneNumber
+    }
+
+    return newContacts;
```

Comment:
Here, you are declaring a variable, giving it a value, and returning it immediately. It might look better if you return the value immediately without creating the `newContacts` variable. Are you able to do it?